### ISSUE TITLE
Repeated log messages related to Notifications when using H2 cache

### ISSUE THREAD

**Issue by john-thomas-dotcms** [dotCMS/core#8439]
Repeated messages are written to the log files about H2 cache issues with notifications.  These messages don't cause any problems with how the system runs, but fill up the logs with unnecessary messages.

**Steps to Reproduce:**
1. Download the dotCMS distribution.
2. Change the cache chain to use the H2 cache provider.
```bash
cache.default.chain=com.dotmarketing.business.cache.provider.timedcache.TimedCacheProvider,com.dotmarketing.business.cache.provider.h2.H2CacheLoader
cache.velocitymemoryonlycache.chain=com.dotmarketing.business.cache.provider.timedcache.TimedCacheProvider
cache.contentletcache.chain=com.dotmarketing.business.cache.provider.timedcache.TimedCacheProvider,com.dotmarketing.business.cache.provider.h2.H2CacheLoader
cache.velocitycache.chain=com.dotmarketing.business.cache.provider.timedcache.TimedCacheProvider,com.dotmarketing.business.cache.provider.h2.H2CacheLoader
```
3\. Startup dotCMS.
4. Login to the server as admin.

**RESULT:**
* Messages similar to the one below are written repeatedly to the log.
	* If no license is installed, the message is written 4 messages every 5 seconds.
	* Once a license is installed, the message frequency is reduced to 1 message every 10-30 seconds.
	* After applying a license and restarting the server, the messages almost all go away (since there are no more notifications).

```text
INFO  2015-12-14 08:19:08,177 com.dotmarketing.business.cache.provider.h2.H2CacheLoader:info(54): returning because object is in cannot cache cache /identifier404cache/identifiercache48190c8c-42c4-46af-8d1a-0cd5db894797-/api/notification/getnewnotificationscount
```

**Notes:**

* Since Notifications are rare, and the messages only display when there are Notifications, the simplest way to see these messages is to start up a new database without a license (since it seems to add some kind of a notification to Try Enterprise).
* These messages only display when using the H2 cache, but bring up the question whether caching notifications may have some other potential issues, or is necessary at all.

**Comment by wezell** [comment]
Bug, good find.  two things - I am not sure why this 404 can't be cached and yes, H2 too noisy ever since  the log4j2 changes for my taste.

**Comment by john-thomas-dotcms** [comment]
Yeah, I think most of the caching has become a little too chatty since the cache changes in 3.3.  We might want to add a card in the next iteration to reduce the cache logging - in most cases it looks like this could be done by just changing the log level of a few messages from INFO to DEBUG.

The good news is that, in the mean time, it looks like almost all the chatty messages can be worked around by adding appropriate Loggers to the log4j2.xml file for each cache provider, with the log level of set to DEBUG.

**Comment by wezell** [comment]
fixed in #9200
